builtins/src/types/numbers.rs

```diff
@@ -61,4 +114,212 @@ mod tests {
         let val = to_i56(7_i32 as i64);
         let _val: i32 = i32::sl_from(&val, vm).expect("Value can be converted to i32");
     }
+
+    #[test]
+    fn test_f56() {
+        /* For each of these test numbers, convert them from f64 to f56 and back and
```

Comment:
in rust functions are documented like this:
```
/// a documentation comment has 3 slashes
/// other fun stuff
fn myfun(f: f64)  {
...
}
```

and for no real good reason, the /* ... */ style comment isn't really used in Rust: https://doc.rust-lang.org/book/ch03-04-comments.html#:~:text=In%20Rust%2C%20the%20idiomatic%20comment,%23!%5B

the compiler seems to support it but we don't use it... for some reason. Because of `cargo fmt` ... I think, Rust devs are a bit more religious about this. EIther way in this context this comment should just be a `///` doc comment above the function because it's useful and it describes what the test does.

Reply:
Thanks for the info! It's funny what you miss when you learn a language through its linter. I'll update with proper comments